### PR TITLE
[Env Vars] adding mql server env variables

### PR DESCRIPTION
**Pull request by belindabennett**
@marcodamore @plypaul to confirm comments are correct for definitions (some wild guesses on a few, especially the first three)

@adamrneary for any formatting feedback!

### CODE REVIEW

**Review by adamrneary — APPROVED** [review]
doc format is right! i presume the content is fine but don't have context there.

File: _posts/mql/guides/mql-server-env-vars.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# The database in the data warehouse where Transform will create tables. This database needs to already exist. ex:
+TFD_MQL_DW_DB=""
+# The schema in the data warehouse where Transform will create tables. This schema needs to already exist. ex: transform_dev
+TFD_MQL_DW_SCHEMA=
+# The username that Transform uses when it connects to the data warehouse used for writing. ex: transform_admin or could be blank
+TFD_MQL_DW_USERNAME=""
+# The password that Transform uses when it connects to the data warehouse used for writing. ex: transform_admin or could be blank
+TFD_MQL_DW_PASSWORD=""
+
+# READING
```
Comment (marcodamore) [inline _posts/mql/guides/mql-server-env-vars.md:32]: One thought about this section is that the vast majority of customers won't have two separate engines for reading/writing I believe. Not sure if we want to break this out into a different section?

**Comment by plypaul** [comment]
I have a PR out that will simplify this - let me post with an update.

File: _posts/mql/guides/mql-server-env-vars.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+TFD_MQL_DW_RO_PORT=
+# The username that Transform uses when it connects to the data warehouse used for reading. ex: transform_admin or could be blank
+TFD_MQL_DW_RO_USERNAME=""
+# The password that Transform uses when it connects to the data warehouse used for reading. ex: transform_admin or could be blank
+TFD_MQL_DW_RO_PASSWORD=""
+
+# METADATA
+# The dialect of the data warehouse used for metadata - ex: livy, presto_hive, redshift, mysql
+TFD_MQL_METADATA_DIALECT=
+# The full qualified domain name of the server that hosts the data warehouse used for metadata. ex: http://livy.staging.testcompany.com/
```
Comment (marcodamore) [inline _posts/mql/guides/mql-server-env-vars.md:47]: I believe we don't want the protocol specified here. I.e. if it's mysql you'd just have the hostname

**Review by marcodamore — APPROVED** [review]
Thanks for doing this. Mostly looks good, just a few comments. 

Longer term thought but not for today: Could we have the MQL server config parameters auto-generate this documentation programmatically, so that when anyone added one, they'd be required to write this info alongside it? It could help keep these in sync

**Comment by adamrneary** [comment]
@marcodamore 100%. See https://docs-pearl-nine.vercel.app/mql/reference/objects for a live example of auto-generated code for all of the GraphQL API surface. Agreed we should be able to figure out how to generate some of these other pieces, as well.

**Comment by plypaul** [comment]
Some updates with the new format: https://github.com/transform-data/docs/pull/8

**Pull request merged**